src/sort-files.js

```diff
@@ -52,8 +52,26 @@ module.exports = function sortFiles(packageJson) {
 
   const isPackageMain = (filepath) => filepath === main;
   const ignored = or(ALWAYS_INCLUDED, ALWAYS_EXCLUDED, isPackageMain);
+  const directoriesFirst = (a, b) => {
+    if (a.endsWith('/') && !b.endsWith('/')) {
+      return -1;
+    } else if (!a.endsWith('/') && b.endsWith('/')) {
+      return 1;
+    } else {
+      return a - b;
+    }
+  };
+  const exclusionsLast = (a, b) => {
```

Comment:
🚨 I'm not sure that it's safe to change the order without potentially changing how the exclusions resolve?

Reply:
I've just seen the original issue you filed. Is there any risk with moving the exclusions to the bottom?

Reply:
The point is that files matched by exclusions that are included again _further down_ the list leads to unintended behavior. That is, the exclusions would **have no effect**. Therefore, exclusions should always be placed last (if they are not needed/wanted, they should be removed).